modules/EnergyManager/Market.cpp

```diff
@@ -245,14 +245,10 @@ Market::Market(types::energy::EnergyFlowRequest& _energy_flow_request, const flo
 
     // Recursion: create one Market for each child
     for (auto& flow_child : _energy_flow_request.children) {
-        _children.emplace_back(flow_child, _nominal_ac_voltage, this);
+        _children.emplace_back(std::make_unique<Market>(flow_child, _nominal_ac_voltage, this));
     }
 }
 
-const std::vector<Market>& Market::children() {
-    return _children;
-}
-
```

Comment:
Is this dead code?  If so, I would appreciate if this either is mentioned in the commit message or goes to a separate commit